src/persist_var.cpp

```diff
@@ -68,17 +68,17 @@ static void get_global_variable(persist_context &ctx, const vconfig &pcfg)
 	config::attribute_value pcfg_side = pcfg["side"];
 	const int side = pcfg_side.to_int(resources::controller->current_side());
 	persist_choice choice(ctx, global, side);
-	config cfg = mp_sync::get_user_choice("global_variable",choice,side).child("variables");
+	config cfg = mp_sync::get_user_choice("global_variable",choice,side).mandatory_child("variables");
```

Comment:
Doesn't it look like this should be an `optional_child`? Given that things were trying to test its validity further down.